src/main/resources/wa-task-configuration-ia-asylum.dmn

```diff
@@ -673,7 +673,7 @@
 "uploadHearingRecording",
 "postHearingAttendeesDurationAndRecording",
 "decideAnFTPA",
-"printAndSendHoBundle"</text>
+"printAndSendHoBundle", "printAndSendHearingRequirements"</text>
```

Comment:
For easier to rebase in the future, can you add this on the next line? Follow the format above